plugins/fastboot/README.md

```diff
@@ -47,6 +47,12 @@ Block size to use for transfers.
 
 Since: 1.2.2
 
+### FastbootOperationDelay
+
+Time in us to delay after a read or write operation.
```

Comment:
I think using ms rather than us is probably more typical for this kind of thing.